### PR TITLE
FSE: Fix typo in contributor usernames

### PR DESCRIPTION
**Pull request by ianstewart**
This PR just fixes a typo for my username in the FSE Plugin. I'm iandstewart in core. :)

### CODE REVIEW

File: apps/full-site-editing/full-site-editing-plugin/readme.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 === Full Site Editing ===
-Contributors: alexislloyd, allancole, automattic, bartkalisz, codebykat, copons, dmsnell, get_dave, glendaviesnz, gwwar, iamtakashi, jeryj, Joen, jonsurrell, kwight, marekhrabe, mattwiebe, mkaz, mmtr86, mppfeiffer, noahtallen, nrqsnchz, nosolosw, obenland, okenobi, owolski, themeshaper, vindl
+Contributors: alexislloyd, allancole, automattic, bartkalisz, codebykat, copons, dmsnell, get_dave, glendaviesnz, gwwar, iamtakashi, jeryj, Joen, jonsurrell, kwight, marekhrabe, mattwiebe, mkaz, mmtr86, mppfeiffer, noahtallen, nrqsnchz, nosolosw, obenland, okenobi, owolski, iandstewart, vindl
```
Comment (vindl) [inline apps/full-site-editing/full-site-editing-plugin/readme.txt:2]: Now it's not in alphabetical order anymore. 😄 
https://gph.is/2fh0nG2

Comment (ianstewart) [reply apps/full-site-editing/full-site-editing-plugin/readme.txt:2]: But do we blame themeshaper or iandstewart for this?

Comment (noahtallen) [reply apps/full-site-editing/full-site-editing-plugin/readme.txt:2]: 🤔 🤔 🤔

Comment (p-jackson) [reply apps/full-site-editing/full-site-editing-plugin/readme.txt:2]: Oops, I'll have to add myself to the list just to fix this up 😆

**Pull request merged**